testing/trino-testing/src/main/java/io/trino/testing/containers/HiveHadoopTestContainer.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.testing.containers;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.net.HostAndPort;
+import org.testcontainers.containers.Network;
+import org.testcontainers.utility.MountableFile;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class HiveHadoopTestContainer
+        extends TestContainer
+{
+    public static final String DEFAULT_IMAGE = "ghcr.io/trinodb/testing/hdp2.6-hive:41";
```

Comment:
why this specific version?

Reply:
I've aligned it with image version used in Product Test. It's also explicit there.
https://github.com/trinodb/trino/blob/1641ae17bf9d477dd1ea9fe4aa2a62f519610072/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentDefaults.java#L18

Reply:
I guess this is fine for now. Eventually, we will probably want to move to `testing/hive3.1-hive` as HDP is EOLing.